vms/platformvm/state/state.go

```diff
@@ -1487,6 +1646,13 @@ func (s *state) AddStatelessBlock(block blocks.Block) {
 }
 
 func (s *state) SetHeight(height uint64) {
+	if s.indexedHeights == nil {
+		s.indexedHeights = &heightRange{
+			LowerBound: height,
+		}
+	}
```

Comment:
could we add a comment on why we set the lower bound regardless of the value in this case?

Reply:
added